.travis.yml

```diff
@@ -4,13 +4,12 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PY=27
-    - CONDA_PY=34
     - CONDA_PY=35
```

Comment:
@scopatz Will I get another automatic PR for that, if all dependencies have been build? Sorry, I'm not up to date on how it woks nowadays and http://conda-forge.github.io/docs/ is silent on that matter.